apps/datahub/src/app/app.component.html

```diff
@@ -33,8 +33,6 @@ <h2 class="text-white text-4xl uppercase">Data Hub</h2>
     <gn-ui-main-search
       *ngIf="(isRecordOpened$ | async) === false"
     ></gn-ui-main-search>
-    <gn-ui-metadata-full-view
-      *ngIf="isRecordOpened$ | async"
-    ></gn-ui-metadata-full-view>
+    <gn-ui-dataset-view *ngIf="isRecordOpened$ | async"> </gn-ui-dataset-view>
```

Comment:
I think this needs some refactoring.
`dataset-view` is the larger container for the metadata view ?